include/logging/log_core.h

```diff
@@ -185,6 +185,49 @@ extern "C" {
 		}							 \
 	} while (false)
 
+static inline log_arg_t __do_strdup(u32_t msk, u32_t idx, log_arg_t param)
+{
+	if (msk & (1<<idx)) {
```

Comment:
nit: spaces around `<<`

Reply:
In general, Zephyr requires spaces around operators so please fix in other places as well.